web/packages/design/src/Image/Image.tsx

```diff
@@ -21,16 +21,35 @@ import styled from 'styled-components';
 import PropTypes from 'prop-types';
 
 import {
-  space,
+  alignSelf,
+  AlignSelfProps,
   color,
-  width,
+  ColorProps,
   height,
-  maxWidth,
+  HeightProps,
   maxHeight,
-  alignSelf,
+  MaxHeightProps,
+  maxWidth,
+  MaxWidthProps,
+  space,
+  SpaceProps,
+  width,
+  WidthProps,
 } from 'design/system';
 
-const Image = props => {
+interface ImageProps
+  extends SpaceProps,
+    ColorProps,
+    WidthProps,
+    HeightProps,
+    MaxWidthProps,
+    MaxHeightProps,
+    AlignSelfProps {
+  alt?: string;
+  style?: React.CSSProperties;
```

Comment:
is `style` required to do something like this?

```ts
<Image style={{css}} />
```

Reply:
Yes. We might avoid listing these properties separately, and instead use the construct that basically says "all allowed attributes of Image element", but I encountered some problems there, and decided it's not worth generalizing if right now, we only use these two.